vic/drivers/shared_image/include/vic_mpi.h

```diff
@@ -27,8 +27,7 @@
 #ifndef VIC_MPI_H
 #define VIC_MPI_H
 
-#include <stddef.h> // used for offsetof() macro
```

Comment:
Now included in `vic_def.h` just to be safe.  I'm guessing that it was added to the standard in C99.